BookAr/src/screens/RegistrationScreen/RegistrationScreen.js

```diff
@@ -13,15 +13,15 @@ export default function RegistrationScreen({navigation}) {
   const [selectedBooks, setSelectedBooks] = useState('')
 
   const genres = [
-    {label: 'Action and Adventure', value: 'The Three Musketeers'},
-    {label: 'Biographies', value: 'Autobiography of a Fat Bride : True Tales of a Pretend Adulthood'},
-    {label: 'Classics', value: 'To Kill a Mockingbird'},
-    {label: 'Fantasy', value: 'The Lord of the Rings'},
-    {label: 'Sci-Fi', value: 'The Handmaid\'s Tale'},
-    {label: 'Suspense and Thrillers', value: ''},
-    {label: 'Cookbooks', value: 'French Chef Cookbook'},
-    {label: 'Poetry', value: 'The Heart of a Woman'},
-    {label: 'I\'m a loser who doesn\'t read', value: 'The Best American Erotica 2004 (Best American Erotica)'},]
+    {label: "Action and Adventure", value: "1569870888"},
+    {label: "Biographies", value: "0553296981"},
+    {label: "Classics", value: "0446310492"},
+    {label: "Fantasy", value: "0345272579"},
+    {label: "Sci-Fi", value: "0449212602"},
+    {label: "Suspense and Thrillers", value: "0316733741"},
+    {label: "Cookbooks", value: "0553233440"},
+    {label: "Poetry", value: "0394512731"},
+    {label: "I'm a loser who doesn't read", value: "0743222628"},]
```

Comment:
At the moment, this uses representative ISBN numbers for each genre. Should we use a different system instead of this?

Reply:
We could possibly use titles instead? I think the risk that a given title isn't in our dataset is lower than if a given ISBN isn't

Reply:
Hey, thanks for the comment - I used titles originally then switched to ISBNs since I thought ISBNs are more consistent, I took these #s from the dataset so they should be there, but I can switch them back if that'll work better?

Reply:
Oh if they're in the dataset then don't worry about it this works